### ISSUE TITLE
Missing indicator for successful install

### ISSUE THREAD

**Issue by cole-h** [DeterminateSystems/harmonic#80]
One thing that could be improved overall is that we never tell the user they may need to reload their environment (by opening a new terminal or `exec $SHELL` or something). If you run it without `-v`, it just silently exits (which is fine, though I would like some indication that it completed successfully by default...), and if you try to `nix-shell` right after that, it won't work and the user is left guessing if it actually completed, or if something went wrong. I'll make a ticket about this.

_Originally posted by @cole-h in https://github.com/DeterminateSystems/harmonic/pull/34#pullrequestreview-1199655057_

---

Basically, I think this issue is twofold:

1. We never notify the user that the install finished successfully
2. We never notify the user that they may need to restart their shell (either by opening a new terminal, `exec $SHELL`, or anything else)

Example screenshot where I fumble through this exact thing:

![image](https://user-images.githubusercontent.com/28582702/204863734-66605a02-2ffe-4753-bc42-bfa3a4992d58.png)
